schemas/regrets-reporter/regrets-reporter-update/regrets-reporter-update.1.schema.json

```diff
@@ -0,0 +1,230 @@
+{
+  "$comment": "See https://docs.google.com/document/d/104x-QmODN7CQwvKVDlB4D2BJzJ0Q-BKnluW5EA7vX7Q/edit",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "AmountsByCategory": {
```

Comment:
I regret to inform you that the tooling used to generate BigQuery tables from jsonschema doesn't support definitions. The convention we use for templating in this case is to define the subset of the schemas under `template/include/<namespace>/<definition>.<version>.schema.json`. You would then call it using cmake includes e.g.

```json
...
        "amount_of_days_of_at_least_one_video_played_on_a_youtube_watch_page": {
          "$ref": "#/definitions/AmountsByCategory"
        },
...
```

would turn into a `templates/include/regrets-reporter/amountsByCategory.1.schema.json`

```
      "properties": {
        "in_total": {
          "type": "number"
        },
        "via_non_search_algorithmic_recommendations_content": {
          "type": "number"
        },
        "via_recommendations_with_an_explicit_query_or_constraint_to_optimize_for": {
          "type": "number"
        },
        "via_search_results": {
          "type": "number"
        }
      },
      "type": "object"
```

And would be called via:

```json
...
        "amount_of_days_of_at_least_one_video_played_on_a_youtube_watch_page": {
          @REGRETS_REPORTER_AMOUNTS_BY_CATEGORY_1_JSON@
        },
...
```

Reply:
Thanks for the heads up. Fixed now :) Btw I found the working syntax to be `@REGRETS-REPORTER_AMOUNTSBYCATEGORY_1_JSON@`